ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/OverwriteAppDialog.java

```diff
@@ -0,0 +1,61 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.view.dialogs;
+
+import org.iplantc.de.admin.desktop.client.toolAdmin.ToolAdminView;
+import org.iplantc.de.admin.desktop.client.toolAdmin.view.subviews.ToolPublicAppListWindow;
+import org.iplantc.de.client.models.IsHideable;
+import org.iplantc.de.client.models.apps.AppAutoBeanFactory;
+import org.iplantc.de.client.models.apps.AppList;
+import org.iplantc.de.commons.client.views.dialogs.IPlantDialog;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.inject.Inject;
+import com.google.web.bindery.autobean.shared.AutoBeanCodex;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.widget.core.client.container.FlowLayoutContainer;
+
+/**
+ * @author aramsey
+ */
+public class OverwriteAppDialog extends IPlantDialog implements IsHideable {
```

Comment:
This dialog should be named `OverwriteToolDialog`, since it's a warning about overwriting a tool and not an app.